src/js/media-captions-menu-button.js

```diff
@@ -180,16 +226,22 @@ class MediaCaptionsMenuButton extends window.HTMLElement {
 
   enable() {
     this.addEventListener('click', this.#handleClick);
+    this.#menuButton.removeAttribute('disabled');
   }
   disable() {
     this.removeEventListener('click', this.#handleClick);
+    this.#menuButton.setAttribute('disabled', '');
```

Comment:
suggestion(non-blocking): might want to hide/collapse the list on disabled if it's open.

Reply:
Going to bump this to subsequent listbox followups.